about.html

```diff
@@ -0,0 +1,107 @@
+                </ul>
+            </div>
+
+        </nav>
+    </header>
+    <main id="main">
+        <section class="intro">
+            <div class="blur aboutBlur">
+                <img src="./assets/images/hamburger.svg" alt="hamburger" class="hamburger" id="humbarger">
+                <span class="welcome"> Global University</span>
+                <h1 class="title">Global University is the most famous University in the Europa</h1>
+                <div class="introDescription aboutIntroDescription">
+                    This year Global University accepts more than 3 thousand of students from out side of the Europa
+                    Our University stublished at 2000 and graduedtd around 1700 students from deffrent faculties.
+                </div>
+                <div class="introFooter introFooterAbout">Please contact us by email for further questions about global University at globaluniversity@gb.org</div>
+            </div>
+        </section>
+        <section class="aboutLogo">
+            <div class="aboutLogoTitle">Global University global summit 2015 logo</div>
+            <div class="line"></div>
+            <div class="aboutLogoDescription">
+                The logo of the global University was designed through the online competition and chosen in online selection
+            </div>
+         <div class="aboutLogoImg">  <img src="./assets/images/logo.png" alt="logo"></div>
+
+        </section>
+        <hr>
+        <section class="previousActivities">
+            <div class="previousActivitiesTitle">See the past graduated students</div>
+            <div class="line"></div>
+            <div class="previosActivitesContainer">
+                <div class="graduate graduate1">
+                    <div>2000</div>
+                    <div>BBA students at Global University</div>
+                </div>
+                <div class="graduate graduate2">
+                    <div>2005</div>
+                    <div>BCA students at Global University</div>
+                </div>
+            </div>
+
+        </section>
+        <section class="partners">
+            <div class="partnersTitle">partners</div>
+            <div class="line3"></div>
+            <div class="sponsors">
+                <img src="./assets/images/partners/amazon.png" alt="amazon">
+                <img src="./assets/images/partners/google.png" alt="google">
+                <img src="./assets/images/partners/invision.png" alt="invision">
+                <img src="./assets/images/partners/toptal.png" alt="toptal">
+                <img src="./assets/images/partners/typeform.png" alt="typeform">
+            </div>
+        </section>
+    </main>
+    <footer class="footer">
+        <div class="footerWrapper">
+            <div>
+                @GlobalUniversity
+            </div>
+            <div>
+                2023 copyright
+            </div>
+        </div>
+    </footer>
```

Comment:
- [ ] While you did a good job styling the `footer` section of the about page :+1:, as a project requirement you should follow the design from [Cindy Shin](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015). So kindly change the background of the footer to be dark, as the template required :+1: 

**Your design**

![Screenshot from 2023-02-17 08-16-56](https://user-images.githubusercontent.com/105117832/219663475-1620660b-9b2a-4d59-84fc-f760c0ab2dc2.png)

**Template**

![Screenshot from 2023-02-17 08-16-37](https://user-images.githubusercontent.com/105117832/219663527-ed48423d-49d9-403e-925c-204df2de3a1d.png)